src/guides/v2.4/test/integration/parameterized_data_fixtures.md

```diff
@@ -0,0 +1,91 @@
+---
+group: testing
+title: Parameterized Data Fixtures
+---
+
+{:.bs-callout-info}
+Parameterized Data Fixtures currently only available for Magento Open Source contributors, and will be released for public with Magento Open Source 2.5.5.
+
+## Overview
+
+Parameterized data fixtures is a scalable and flexible way to create a data fixture for integration tests. Such fixtures can accept parameters directly from annotations, like, but not limited to, entity name, status, quantity and so on,
+therefore eliminating the need of creating almost identical fixtures every time a slight change is needed.
+
+### Benefits
+
+Parameterized data fixtures allows developers to:
+
+-  Reduce duplicated codes in integration fixture files.
+-  Reduce the number of fixture files.
+-  Remove fixtures coupling.
+-  Write reusable and customizable data fixtures.
+-  Focus on test scenarios and not fixtures.
+-  Save time in development and code review process.
+
+### Decoupled fixtures class examples
+
+Example 1:
+
+```php?start_inline=1
+class QuoteTest extends \PHPUnit\Framework\TestCase
+{
+   /**
+   * @magentoDataFixture \Magento\Catalog\Fixture\CreateSimpleProduct with:{"sku": "simple1", "price": 5.0} as:product1
+   * @magentoDataFixture \Magento\Catalog\Fixture\CreateSimpleProduct with:{"sku": "simple2", "price": 10.0} as:product2
+   * @magentoDataFixture \Magento\Quote\Fixture\CreateEmptyCartForGuest as:cart
+   * @magentoDataFixture \Magento\Quote\Fixture\AddSimpleProductToCart with:{"cart": "$cart", "product": "$product1", "qty": 2}
+   * @magentoDataFixture \Magento\Quote\Fixture\AddSimpleProductToCart with:{"cart": "$cart", "product": "$product2", "qty": 1}
+   */
```

Comment:
Let's use this example https://github.com/magento-l3/magento2ce/blob/5a0188ec94abdbbf066a155767f404a1c348635a/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Guest/CartTotalsTest.php#L196-L202